frontend/src/features/admin-form/settings/SettingsPage.tsx

```diff
@@ -51,6 +56,18 @@ export const SettingsPage = (): JSX.Element => {
       navigate(`${ADMINFORM_ROUTE}/${formId}/${ADMINFORM_RESULTS_SUBROUTE}`)
   }, [formId, hasEditAccess, isCollabLoading, navigate])
 
+  const multiLangTab =
+    isUserLoading ||
+    isFormSettingLoading ||
+    !user?.betaFlags?.multiLangTranslation
+      ? null
+      : {
+          label: 'Multi-language',
+          icon: LanguageTranslation,
+          component: SettingsMultiLangPage,
+          path: 'language',
+        }
```

Comment:
/nit it is easier to read the inverse.

```suggestion
    !isUserLoading &&
    !isFormSettingLoading &&
    user?.betaFlags?.multiLangTranslation
      ? {
          label: 'Multi-language',
          icon: LanguageTranslation,
          component: SettingsMultiLangPage,
          path: 'language',
        }
      : null
```